rudder-agent/SOURCES/Makefile

```diff
@@ -316,6 +332,11 @@ build-lmdb: lmdb-source
 	cd build-lmdb/opt/rudder && mkdir -p bin lib include man/man1
 	cd lmdb-source/libraries/liblmdb && $(MAKE) install prefix=$(RUDDER_DIR) DESTDIR=$(CURDIR)/build-lmdb
 
+pcre-lmdb: pcre-source
+	cd pcre-source && ./configure --disable-cpp --enable-utf8 --enable-unicode-properties --prefix=$(RUDDER_DIR)
+	cd pcre-source && $(MAKE)
+	cd pcre-source && $(MAKE) install DESTDIR=$(CURDIR)/build-pcre
```

Comment:
Why do you do the make install in the build target, not the install target below?